options/intl/generic/libintl-stubs.cpp

```diff
@@ -2,49 +2,59 @@
 #include <bits/ensure.h>
 
 char *gettext(const char *msgid) {
+	return (char *)msgid;
+	return nullptr;
 	__ensure(!"Not implemented");
 	__builtin_unreachable();
 }
```

Comment:
Huh? Why do we return nullptr here (and in the other functions)? Why are there two returns?